src/main/resources/application.yaml

```diff
@@ -45,6 +45,10 @@ s2s:
   url: ${S2S_URL:}
   useStub: ${S2S_USER_STUB:false}
 
+maxAge:
+  default: ${MAX_AGE_DEFAULT:14}
```

Comment:
is there any reason for this default?
I would have thought at least a month, probably longer, merging this will get deployed in our draft store we deploy so need to think about either the default or we just set it for a long time

Reply:
This change does not affect old api, drafts are safe.

The default max age value is open for discussion :)